src/features/Character/queries/config.ts

```diff
@@ -0,0 +1,12 @@
+/** File with default configurations for queries in `Character` feature. */
+
+/** 10 minutes. */
+export const defaultStaleTime = 10000 * 60;
+
+/** Default fetch strategy for queries.  */
+export const defaultFetchStrategy = {
+  retry: false,
+  refetchOnMount: false,
+  refetchOnWindowFocus: false,
+  refetchOnReconnect: false,
```

Comment:
I don't like that this config just the same as in `VisualNovel` feature, maybe it would be to create some global config which features would extend if needed?